### PR TITLE
Fixes #564 and restore color changing logo

### LINKED ISSUE #564: Can't read properties of null error upon opening the website
When I am opening the website, I get this error. 

Screenshot (478)

### PR DESCRIPTION
**Pull request by therealharshit**
I have restored color changing logo which also fixes #564.
@quozl please review.

### CODE REVIEW

File: css/custom.css [hunk 1]
```diff
@@ -15,3 +15,8 @@
 .intro_img>.container>div>div>div>p {
   padding: 0 10px;
 }
+
+/*This is for navLogo of navbar */
+#navLogo {
+  margin-top: 6px;
+}
```
Comment (quozl) [inline css/custom.css:22]: Missing newline at end of file.

**Comment by quozl** [comment]
Commit message is very brief; can we have more detail for history?  See [our guidance](https://github.com/sugarlabs/sugar-docs/blob/master/src/contributing.md#making-commits) in particular;
* [ ] we're restoring a feature that got broken, so not "add",
* [ ] explain the problem,
* [ ] reference what commit broke the feature.

**Comment by therealharshit** [comment]
> Commit message is very brief; can we have more detail for history? See [our guidance](https://github.com/sugarlabs/sugar-docs/blob/master/src/contributing.md#making-commits) in particular;
> 
> * [x]  we're restoring a feature that got broken, so not "add",
> * [x]  explain the problem,
> * [x]  reference what commit broke the feature.

@quozl I have made all the suggested changes to the commit messages, please review.

**Comment by quozl** [comment]
* merged your missing newline commit as it was a problem you introduced within the first commit,
* rewrote the commit message following our guidance; the problem was not explained, and there was no reference to the commit that broke the feature (only a reference to a pull request, which isn't available at the time git is used offline),

**Comment by therealharshit** [comment]
> * squashed your missing newline commit as it was a problem you introduced within the first commit,
> * rewrote the commit message following our guidance; the problem was not explained, and there was no reference to the commit that broke the feature (only a reference to a pull request, which isn't available at the time git is used offline),

Thanks @quozl I'll make sure to follow the guidelines while writing the commit messages in future.

**Pull request merged**